README.md

```diff
@@ -116,14 +117,16 @@ params.adjustments.fajr = 2
 | kuwait | Method used by the country of Kuwait. Fajr angle: 18, Isha angle: 17.5 |
 | moonsightingCommittee | Moonsighting Committee. Fajr angle: 18, Isha angle: 18. Also uses seasonal adjustment values. |
 | singapore | Method used by Singapore. Fajr angle: 20, Isha angle: 18. |
+| tehran | Institute of Geophysics, University of Tehran. Fajr angle: 17.7, Maghrib angle: 4.5, Isha angle: 14. |
+| shia | Shia Ithna-Ashari, Leva Institute, Qum. Fajr angle: 16, Maghrib angle: 4, Isha angle: 14. |
```

Comment:
The tehran calculation method makes perfect sense because it's specific angle that is valid for a specific location. I have reservations about including a method with extremely different values (isha angle of 14 is a very different value), without it being for a specific region. I think right now it would be best if we only added the tehran method.